release/bin/macos/rttr.command

```diff
@@ -3,16 +3,17 @@
 cd $(dirname $0)
 
 if [ ! "$1" = "interminal" ] ; then
+	scriptPath=$(pwd)
 	echo "#!/bin/sh" > /tmp/rttr.command
-	echo "$0 interminal" >> /tmp/rttr.command
+	echo "$scriptPath/$0 interminal" >> /tmp/rttr.command
```

Comment:
I meant when you call it via command line. Then `$0` gets only the relative path